test/system/src/main/java/io/pravega/test/system/framework/services/kubernetes/ZookeeperK8sService.java

```diff
@@ -77,8 +77,8 @@ public boolean isRunning() {
                                                       .filter(podStatus -> podStatus.getContainerStatuses()
                                                                                     .stream()
                                                                                     .allMatch(st -> st.getState().getRunning() != null))
-                                                      .count())
-                        .thenApply(runCount -> runCount == DEFAULT_INSTANCE_COUNT)
+                                                      .count() * 2 - statuses.size())
```

Comment:
Why do we need this change?

Reply:
See: https://github.com/pravega/pravega/pull/5757#discussion_r608818218